Program.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.EntityFrameworkCore;
+using WsuTriangle.AttendanceTracker;
+
+var builder = WebApplication.CreateBuilder(args);
+
+#region Services
+var services = builder.Services;
+services.AddControllers();
+services.AddEndpointsApiExplorer();
+services.AddSwaggerGen();
+
+services.AddDbContext<AppDbContext>(options => options
+    .UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"), opt => opt
+        .EnableRetryOnFailure()
+        .UseQuerySplittingBehavior(QuerySplittingBehavior.SplitQuery)
+        )
+);
+
+#endregion Services
```

Comment:
Here is where you will add authentication.

https://learn.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-8.0